cmake/ev-cli.cmake

```diff
@@ -15,11 +12,33 @@ macro(setup_ev_cli)
         if(NOT ${IS_PYTHON_VENV_ACTIVE})
             message(FATAL_ERROR "Python venv is not active. Please activate the python venv before running this command.")
         endif()
-        set(EV_CLI "${${PROJECT_NAME}_PYTHON_VENV_PATH}/bin/ev-cli")
-        add_dependencies(ev-cli
-            ev-dev-tools_pip_install_dist
+
+        find_program(EV_GET_PACKAGE_LOCATION
+            NAMES get_package_location.py
+            PATHS "${EVEREST_SCRIPTS_DIR}"
+            NO_DEFAULT_PATH
         )
+        execute_process(
+            COMMAND ${Python3_EXECUTABLE} ${EV_GET_PACKAGE_LOCATION} --package-name ev-dev-tools
+            OUTPUT_VARIABLE EV_DEV_TOOLS_PACKAGE_LOCATION
+            OUTPUT_STRIP_TRAILING_WHITESPACE
+            RESULT_VARIABLE
+            EV_GET_PACKAGE_LOCATION_RESULT
+        )
+
+        message(STATUS "Using ev-dev-tool package: ${EV_DEV_TOOLS_PACKAGE_LOCATION}")
+
+        find_program(EV_CLI ev-cli PATHS ${EV_ACTIVATE_PYTHON_VENV_PATH_TO_VENV}/bin REQUIRED)
```

Comment:
Just figured that out (see https://cmake.org/pipermail/cmake/2010-October/040460.html), that `PATHS` items are search after system default path (in my case, my local ev-cli installation got found all the time, that's probably not, what we want in the venv scenario), whereas `HINTS` items are searched before.
So instead of `PATHS` we should probably use `HINTS` here.

Another issue here could be that `find_program` creates a cached variable, and hence, once it succeeded it will always use the same found path.  Still I like `find_program` more than hardcoding  (as it was before).
The case where developers might want to change from one venv to another or so, they would need to delete the `EV_CLI` var from `CMakeCache.txt` so that `find_program` finds the new correct one.  Not sure what to do here (we could also unset it every time in order to force `find_program` to search for it again?)